nntrainer/graph/network_graph.cpp

```diff
@@ -457,6 +457,53 @@ void NetworkGraph::setBatchSize(unsigned int batch_size) {
     allocateTensors(exec_mode);
 }
 
+void NetworkGraph::tryZeroGrad(LayerNode *node) {
+  auto &rc = node->getRunContext();
+  auto current_execution_order = std::get<1>(node->getExecutionOrder());
```

Comment:
can you rename this to calc_grad_exec_order?

Reply:
Updated